pyenv-win/bin/pyenv.bat

```diff
@@ -22,6 +22,19 @@ set "bindir="
 set "extrapaths="
 for /f "%skip_arg%delims=" %%i in ('%pyenv% vname') do call :extrapath "%~dp0..\versions\%%i"
 
+:: Add %AppData% Python Scripts to %extrapaths%.
+for /F "tokens=1,2 delims=-" %%i in ('%pyenv% vname') do (
+  if /i "%%j" == "win32" (
+    for /F "tokens=1,2,3 delims=." %%a in ("%%i") do (
+        set "extrapaths=%extrapaths%%AppData%\Python\Python%%a%%b-32\Scripts;"
```

Comment:
is this specific to python 3.2?

Reply:
`-32` means 32bit version of Python. For example, when a user installed 3.10.4-win32 for a project, pip packages will be installed inside `%AppData%\Python\Python310-32`.

Reply:
64bit version at `%AppData%\Python\Python310`